vector/v.random/testsuite/test_v_random.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+output = 'test01'
+output2 = 'test02'
+npoints = 100
+state = 'boundary_state'
+zmin=10
+zmax=120
+
+class Test_v_random(TestCase):

```

Comment:
There is no ideal way of doing this, but to comply with PEP8, use TestVRandom as the name of the class. Something like BasicTest would be fine too as it is in a well named file in a v.random directory.